Missing favicon on login and register

There is no favicon link on login and register pages. Most browser then try to call `favicon.ico`, which does not exist, and creating errors in logs:
```
powerdnsadmin             | [2020-05-09 21:44:44 +0000] [22] [ERROR] Error handling request /favicon.ico
powerdnsadmin             | Traceback (most recent call last):r="(&(objectClass=PostfixBookMailAccount)(uid=erpnext))"
powerdnsadmin             |   File "/usr/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_requestry
powerdnsadmin             |     rv = self.dispatch_request()
powerdnsadmin             |   File "/usr/lib/python3.8/site-packages/flask/app.py", line 1925, in dispatch_requestr="(&(objectClass=PostfixBookMailAccount)(uid=erpnext))"
powerdnsadmin             |     self.raise_routing_exception(req)
powerdnsadmin             |   File "/usr/lib/python3.8/site-packages/flask/app.py", line 1907, in raise_routing_exception
powerdnsadmin             |     raise request.routing_exceptionr="(&(objectClass=PostfixBookMailAccount)(uid=erpnext))"
powerdnsadmin             |   File "/usr/lib/python3.8/site-packages/flask/ctx.py", line 350, in match_requestry
powerdnsadmin             |     result = self.url_adapter.match(return_rule=True)
powerdnsadmin             |   File "/usr/lib/python3.8/site-packages/werkzeug/routing.py", line 1945, in matchr="(&(objectClass=PostfixBookMailAccount)(uid=erpnext))"
powerdnsadmin             |     raise NotFound()
powerdnsadmin             | werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you
 entered the URL manually please check your spelling and try again.
```

--- a/powerdnsadmin/templates/login.html
+++ b/powerdnsadmin/templates/login.html
@@ -5,6 +5,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>Log In - {{ SITE_NAME }}</title>
+  <link rel="icon" href="/static/img/favicon.png">
   <!-- Tell the browser to be responsive to screen width -->
   <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
   {% assets "css_login" -%}

--- a/powerdnsadmin/templates/register.html
+++ b/powerdnsadmin/templates/register.html
@@ -5,6 +5,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>Register - {{ SITE_NAME }}</title>
+  <link rel="icon" href="/static/img/favicon.png">
   <!-- Tell the browser to be responsive to screen width -->
   <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
   {% assets "css_login" -%}

